back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -400,6 +384,24 @@ def invalid_blank_author_for_update?(input, params)
 
     input.participation_method_on_creation.sign_in_required_for_posting?
   end
+
+  # Change counts on idea for values for phase, if filtered by phase
+  def update_phase_voting_counts(ideas, params)
```

Comment:
I think I would have a slight preference to move this logic to the serializer, where the phase could be passed as a parameter from controller to serializer (the ideas_phases would have to be included for index for performance).

Or we could introduce new attributes like `context_baskets_count` and `context_votes_count`, so that we can still get the total counts from the serializer.